content/250-postgres/900-troubleshooting.mdx

```diff
@@ -53,3 +53,34 @@ To remove this warning and generate Prisam Client without the query engine, you
 ```
 npx prisma generate --no-engine
 ```
+
+## Plan limit reached when running `prisma init --db`
+
+### Problem
+
+When running the command:
+
+```terminal
+npx prisma@latest init --db
+```
+
+You may encounter the following error message in your logs:
+
+```
+Workspace plan limit reached for feature "Project".
+```
+
+### Cause
+
+Your default [workspace](/platform/about#workspace) project limit has been reached.
+
+### Solution
+
+To resolve this issue, consider the following options:
+
+1. Configure a different Workspace as your default—one that has available capacity for additional projects.
+1. Delete unused projects or databases from your current default Workspace to free up space.
+1. Ensure that you are logged into the correct account in the Prisma CLI. For more details on authentication and account management, please refer to the [Prisma CLI documentation](/platform/platform-cli/commands#authentication).
+1. [Upgrade to a plan](/platform/about#billing) that supports more projects in your default Workspace.
```

Comment:
are these options or steps? if options, use regular bullets. if steps, use numbered bullets. (it looks like options to me)